### PR TITLE
feat: add "edge" CLI flag to install nuxt-edge

### LINKED ISSUE #86: Nuxt-Edge as selection
### What problem does this feature solve?

I'd love to see an option to create the Nuxt app with `nuxt-edge` instead of `nuxt` :speak_no_evil: 

<div align="right"><sub><em>This feature request is available on <a href="https://cmty.app/nuxt">Nuxt</a> community (<a href="https://cmty.app/nuxt/create-nuxt-app/issues/c62">#c62</a>)</em></sub></div>

### PR DESCRIPTION
**Pull request by NicoPennec**
resolve #86

### CODE REVIEW

File: sao.js [hunk 1]
```diff
@@ -95,6 +98,9 @@ module.exports = {
       default: 'npm'
     }
   },
+  data: {
+    edge: Boolean(argv.edge)
```
Comment (pi0) [inline sao.js]: Can't we simply use `process.argv.includes('--edge')`? :D

Comment (NicoPennec) [reply sao.js]: You right!

Never forget the [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) 😅

**Comment by NicoPennec** [comment]
ping @Atinux (for approval ^^)

**Comment by Atinux** [comment]
Awesome work @NicoPennec!
Ping me on discord instead of GitHub :)

**Pull request merged**